Marlin/src/pins/stm32f4/pins_RUMBA32_common.h

```diff
@@ -143,4 +166,17 @@
     #define LCD_PINS_D7                     PE15
   #endif
 
-#endif
+  // Alter timing for graphical display
+  #if HAS_GRAPHICAL_LCD
+    #ifndef BOARD_ST7920_DELAY_1
+      #define BOARD_ST7920_DELAY_1 DELAY_NS(96)
+    #endif
+    #ifndef BOARD_ST7920_DELAY_2
+      #define BOARD_ST7920_DELAY_2 DELAY_NS(48)
+    #endif
+    #ifndef BOARD_ST7920_DELAY_3
+      #define BOARD_ST7920_DELAY_3 DELAY_NS(600)
+    #endif
+  #endif
+
+#endif
```

Comment:
The Linter says we need an EOL at the end of the file.

Reply:
Should be fixed now.